components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -86,16 +89,31 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 				log.Error(err, "SetReadySubscription() failed for the subscription:", "subscription", subscription)
 				reconcileErr = err
 			} else {
+				//TODO READY Subscription
```

Comment:
Please add an explanation for each `TODO` so that someone else can also understand what needs to be done here.

Reply:
will remove it, remained by mistake